app/src/main/kotlin/com/wire/android/ui/userprofile/other/bottomsheet/OtherUserBottomSheetState.kt

```diff
@@ -10,9 +10,13 @@ class OtherUserBottomSheetState {
 
     private var conversationSheetContent: ConversationSheetContent? by mutableStateOf(null)
     private var groupState: OtherUserProfileGroupState? by mutableStateOf(null)
+    private var flag: OtherUserBottomSheetContentFlag = OtherUserBottomSheetContentFlag.NONE
```

Comment:
I think that `flag` might be too generic for the naming here, maybe something like `OtherUserBottomSheetMode` or `OtherUserContentType` could be a bit better, but either way maybe we can come up with a better name :D

Reply:
as far as it's private var and it supposes to be used only in here, I guess the name `contentFlag` is good enough